omnibus/config/templates/datadog-agent/systemd.process.service.erb

```diff
@@ -1,16 +1,15 @@
 [Unit]
-Description="Datadog Agent"
-Requisite=datadog-agent.service
-After=network.target datadog-agent.target
-PartOf=datadog-agent.service
+Description="Datadog Process Agent"
+After=network.target datadog-agent.service
```

Comment:
As we depend on `datadog-agent.service` who's depending on `network.target` as well, is it still needed ?

Reply:
probably not. Not sure if it's worth removing though. @truthbk WDYT?

Reply:
It doesn't hurt to have an explicit dependency, even if its a NOP, because if the customer does indeed manipulate the service unit, this will keep it honest.